ui/src/app/pages/login/account-creation/account-creation.tsx

```diff
@@ -76,6 +76,18 @@ const researchPurposeList = [
 
 const nameLength = 80;
 
+export const FORM_LABELS = {
```

Comment:
Great idea. Not clear to me why this wouldn't be:
```suggestion
export const formLabels = {
```

Reply:
Because I don't care either way, and the linter didn't stop me 😉

Updated.